website/docs/r/app_service.html.markdown

```diff
@@ -0,0 +1,116 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_app_service"
+sidebar_current: "docs-azurerm-resource-app-service-x"
+description: |-
+  Manages an App Service (within an App Service Plan).
+
+---
+
+# azurerm\_app\_service
```

Comment:
we don't need the \ - this is a fallacy about markdown :)

Reply:
updated